proxy/common/flags.py

```diff
@@ -9,31 +9,38 @@
     :license: BSD, see LICENSE for more details.
 """
 import abc
-import logging
-import importlib
-import collections
 import argparse
 import base64
+import collections
+import importlib
+import inspect
 import ipaddress
+import logging
+import multiprocessing
 import os
+import pathlib
 import socket
-import multiprocessing
 import sys
-import inspect
-import pathlib
-
-from typing import Optional, Union, Dict, List, TypeVar, Type, cast, Any, Tuple
+from typing import Any, Dict, List, Optional, Tuple, Type, TypeVar, Union, cast
 
-from .utils import text_, bytes_
-from .constants import DEFAULT_LOG_LEVEL, DEFAULT_LOG_FILE, DEFAULT_LOG_FORMAT, DEFAULT_BACKLOG, DEFAULT_BASIC_AUTH
-from .constants import DEFAULT_TIMEOUT, DEFAULT_DEVTOOLS_WS_PATH, DEFAULT_DISABLE_HTTP_PROXY, DEFAULT_DISABLE_HEADERS
-from .constants import DEFAULT_ENABLE_STATIC_SERVER, DEFAULT_ENABLE_EVENTS, DEFAULT_ENABLE_DEVTOOLS
-from .constants import DEFAULT_ENABLE_WEB_SERVER, DEFAULT_THREADLESS, DEFAULT_CERT_FILE, DEFAULT_KEY_FILE
-from .constants import DEFAULT_CA_CERT_DIR, DEFAULT_CA_CERT_FILE, DEFAULT_CA_KEY_FILE, DEFAULT_CA_SIGNING_KEY_FILE
-from .constants import DEFAULT_PAC_FILE_URL_PATH, DEFAULT_PAC_FILE, DEFAULT_PLUGINS, DEFAULT_PID_FILE, DEFAULT_PORT
-from .constants import DEFAULT_NUM_WORKERS, DEFAULT_VERSION, DEFAULT_OPEN_FILE_LIMIT, DEFAULT_IPV6_HOSTNAME
-from .constants import DEFAULT_SERVER_RECVBUF_SIZE, DEFAULT_CLIENT_RECVBUF_SIZE, DEFAULT_STATIC_SERVER_DIR
-from .constants import DEFAULT_ENABLE_DASHBOARD, COMMA, DOT
+from .constants import (COMMA, DEFAULT_BACKLOG, DEFAULT_BASIC_AUTH,
```

Comment:
Can you configure your editor to not modify a line of code that wasn't changed?  Otherwise it becomes harder to review between a) cosmetic changes b) actual logic changes I generally don't like to fiddle with import orders (may be it's just your editor).

Reply:
I have configure my editor (atom) with default python conf. I will change it that it will doesnt change the import orders.